src/shared/components/HomeButton.js

```diff
@@ -5,16 +5,25 @@ const useStyles = makeStyles({
     button: {
         borderRadius: "24px",
         textTransform: "none",
-        width: "22%",
+        width: "27%",
         background: "linear-gradient(to left, #7D0202, #7D0202)",
         color: "white",
-        height: "2.8rem",
-        paddingTop: "0.3rem",
+        height: "2.5rem",
+        padding: "0",
         transition: "background 300ms ease-out",
         "&:hover": {
             background: "linear-gradient(to left, #7D0202, #B25F4C)",
         },
-        marginTop: "0.5rem"
+        marginTop: "0.5rem",
+        "@media (min-width: 1500px)": {
```

Comment:
Any reasoning or sources for using this screen width?

Reply:
tailwind's 2xl is 1536. will change to that.

Reply:
Going to change this to Tailwind's '2xl', which is 1536p

Reply:
Okayy